docs/auth/README.adoc

```diff
@@ -1,6 +1,8 @@
 = AeroGear Services Auth SDK
 
-Mobile authentication SDK based on Keycloak
+Mobile authentication SDK based on link:http://www.keycloak.org/[Keycloak] using link:http://openid.net/connect/[OpenID Connect].
+
+Provides authentication features like access control and two factor authentication through Keycloak.
 
 == Adding dependency
 
```

Comment:
[Optional] I have added this section without checking that works. We will need probably bom update to enable it. Worth check

Reply:
good catch. @aidenkeating  see https://github.com/aerogear/aerogear-android-sdk/pull/52